### PR TITLE
docs: update getting started real device

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

Adds section on how to run on a real device

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -63,6 +63,29 @@ yarn android
...
+yarn open-xcode
+```
+
+3. On the left side select Energy and then select the Energy target. On the right side select your device from the dropdown menu.
+
+4. Select Signing & Capabilities and check Automatically manage signing and select your team `Art.sy Inc.` from the dropdown menu.
+
+5. Run the app pressing the play button
+
+> :warning: This will generate a diff on `project.pbxproj` file that you should **not** commit with your changes.
```
Comment (damassi) [inline docs/getting_started.md:87]: 👍

File: docs/getting_started.md [hunk 2]
```diff
@@ -63,6 +63,29 @@ yarn android
...
+yarn install:all
+```
+
+2. Open xcode
+
+```sh
+yarn open-xcode
+```
+
+3. On the left side select Energy and then select the Energy target. On the right side select your device from the dropdown menu.
```
Comment (damassi) [inline docs/getting_started.md:81]: Whats the name of the screen where you select energy / is there a way to get here from the top nav file / edit / view, etc ui? (Not sure what this or the target is)

Comment (gkartalis) [reply docs/getting_started.md:81]: Good point maybe a screenshot would be better or more descriptive

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **It seems like you added some non WebP images to Eigen, please convert them to WebPs using `source images/script.sh` script **
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2a4db8e3e35d0d92c4aba75c6cd9477dd9d2ddc0
</p>

**Pull request merged**